README.md

```diff
@@ -1,32 +1,48 @@
+| pylint.severity         | `{ "convention": "Information", "error": "Error", "fatal": "Error", "refactor": "Hint", "warning": "Warning", "info": "Information" }` | Mapping of Pylint's message types to VS Code's diagnostic severity levels as displayed in the Problems window. You can also use it to override specific Pylint error codes. E.g. `{ "convention": "Information", "error": "Error", "fatal": "Error", "refactor": "Hint", "warning": "Warning", "W0611": "Error", "undefined-variable": "Warning" }`                                                                                                                                                                                                                                        |
+| pylint.path             | `[]`                                                                                                                                   | Path or command to be used by the extension to lint Python files with Pylint. Accepts an array of a single or multiple strings. If passing a command, each argument should be provided as a separate string in the array. Note: Using this option may slowdown linting. <br>Examples: <br>- `["~/global_env/pylint"]` <br>- `["conda", "run", "-n", "lint_env", "python", "-m", "pylint"]`                                                                                                                                                                                                 |
+| pylint.interpreter      | `[]`                                                                                                                                   | Path to a Python executable or a command that will be used to launch the Pylint server and any subprocess. Accepts an array of a single or multiple strings.  When set to `[]`, the extension will use the path to the selected Python interpreter. If passing a command, each argument should be provided as a separate string in the array.                                                                                                                                                                                                                                              |
+| pylint.importStrategy   | `useBundled`                                                                                                                           | Defines which Pylint to be used to lint Python files. When set to `useBundled`, the extension will use the Pylint binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the Pylint binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid Pylint binary in the selected environment, it will fallback to using the Pylint binary that is shipped with the extension The `pylint.path` setting may be ignored when this setting is set to `fromEnvironment`. |
```

Comment:
```suggestion
| pylint.importStrategy   | `useBundled`                                                                                                                           | Defines which Pylint to be used to lint Python files. When set to `useBundled`, the extension will use the Pylint binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the Pylint binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid Pylint binary in the selected environment, it will fallback to using the Pylint binary that is shipped with the extension. The `pylint.path` setting may be ignored when this setting is set to `fromEnvironment`. |
```
`pylint.path` overrides everything, there may be no fallback when this is used.

Reply:
Made an update to try to clarify that